art-admin/internal/store/redis/metadata_test.go

```diff
@@ -31,8 +31,14 @@ func TestMetadata(t *testing.T) {
 		is.NoErr(err)
 	}
 
-	all, err := ms.GetAllOffchainMetadata(ctx)
-	is.NoErr(err)
-	is.Equal(len(all), 100)
+	defer func() {
```

Comment:
defer statements delay the execution of the function until the nearby function returns. So you have to invoke this anon func before end of this func in current case defer make no sense